bookshelf/1-cloud-run/pom.xml

```diff
@@ -96,6 +96,12 @@ Copyright 2019 Google LLC
       <version>2.6</version>
     </dependency>
 
+    <dependency>
```

Comment:
this was transitively pulled in as part of the selenium dependencies (which weren't scoped at `test` previously).

Had to add it back in so it could be used in `BookshelfContextListener`.